.github/workflows/paig-securechat-release.yml

```diff
@@ -2,6 +2,15 @@ name: paig-securechat upload to PyPI and DockerHub
 
 on:
   workflow_dispatch: # Allows manual trigger
+    inputs:
+      skip_publish_to_pypi:
+        description: 'Skip publish to PyPI'
+        required: false
+        default: 'false'
+      skip_publish_to_docker:
+        description: 'Skip publish to Docker Hub'
+        required: false
+        default: 'false'
```

Comment:
can we do 
publish_to_pypi - default true
publish_to_docker -  default true

more easier to read

Reply:
These options will be available on UI while triggering manual workflow . 
Sometimes due to error it may be possible that docker is release and pypi got into error or vice versa .Therefor we can skip publish in such cases using these flags